### DISCUSSION TITLE
is there some way to chat with the maintainers/contributors of this repo?

### DISCUSSION THREAD

**Discussion by xddq** [xmppjs/xmpp.js#905] [Q&A]
I'm literally just curious and would ask a lot of questions :D 

my potential long term goal: creating an open source ios, android (react native), desktop and web (pwa) xmpp client. Targeting less tech savvy users. So far I did only stumble upon AstraChat, which seems to be closed source.

Sorry for the spam, and thank you for the answer, if I get any.

**Comment by sonnyp** [comment]
Feel free to ask here.
